src/xenia/ui/imgui_drawer.cc

```diff
@@ -135,14 +134,15 @@ void ImGuiDrawer::SetupFont() {
 
   // TODO(benvanik): jp font on other platforms?
   // https://github.com/Koruri/kibitaki looks really good, but is 1.5MiB.
-  const char* jp_font_path = "C:\\Windows\\Fonts\\msgothic.ttc";
-  if (xe::filesystem::PathExists(jp_font_path)) {
+  std::filesystem::path jp_font_path = "C:\\Windows\\Fonts\\msgothic.ttc";
+  if (std::filesystem::exists(jp_font_path)) {
     ImFontConfig jp_font_config;
     jp_font_config.MergeMode = true;
     jp_font_config.OversampleH = jp_font_config.OversampleV = 1;
     jp_font_config.PixelSnapH = true;
     jp_font_config.FontNo = 0;
-    io.Fonts->AddFontFromFileTTF(jp_font_path, 12.0f, &jp_font_config,
+    io.Fonts->AddFontFromFileTTF(jp_font_path.u8string().c_str(), 12.0f,
```

Comment:
Since we can't control the path in this situation, I think it would be better to leave it as a `const char*` to avoid the dance with a `std::filesystem::path` object.

Reply:
Ok. That's done.